packages/subiquity_test/lib/src/utils.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:io';
 
-import 'package:subiquity_test/subiquity_test.dart';
+import '../subiquity_test.dart';
```

Comment:
How do you feel about using import paths relative to the package root, like we do in app-center (i.e. `import '/subiquity_test.dart'`)? It makes moving files around within a package much cleaner - does it have any disadvantages?

Reply:
Usually people choose between package imports or relative (never seen anyone using absolutes), we could use package imports for local ones too to accommodate for that. That's what we do in the flame repo.

There is a linter rule for that too.

Reply:
Ah, didn't know that was possible, I like that!

Reply:
https://github.com/canonical/ubuntu-flutter-plugins/pull/367

Reply:
Hm sorry, I think I misunderstood your comment I thought there was a linter rule for those absolute path imports now (`prefer_relative_imports` doesn't complain about those, but the auto-fix options only offer package imports or relative ones). I'm not sure if it's worth converting all imports to package imports, since we've been using relative ones mostly. What do you prefer?

Reply:
Ok, I see. If you've had good experience with package imports I'd also be fine using those. (Let's hope we won't have to rename packages too often again :D)